File: NETMouse - .NET release/Examples/C#/Arrays_7.cs
using ABCNET.Extensions;
using ABCNET.Utils;
using System.Linq;

namespace TestProgram
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            int t = Base.ReadInteger("T:");
            Arr.Rand(Base.ReadInteger("N:"), -56, 47).Println()
                .Where((x, i) => (i % 2 == 0) && (x > t)).Aggregate((a, b) => a * b).Println();
        }
    }
}

File: NETMouse - .NET release/Examples/PascalABC.NET/Arrays_7.pas
{$reference ABCNET.dll}
uses ABCNET.Utils;
uses ABCNET.Extensions;
begin
  var t := BaseU.ReadInteger('T:');
  Arr.Rand(BaseU.ReadInteger('N:'), -56, 47).Println()
      .Where((x, i) -> (i mod 2 = 0) and (x > t)).Aggregate((a, b) -> a * b).Println();
  BaseU.ReadChar();
end.